rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+
+* **New AWS roles supporting the ROSA UI**: The tools to manage the `ocm-role` and `user-role` and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The `ocm-role` and `user-role` are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
+
+[id="rosa-documentation-enhancements"]
+=== Documentation enhancements
+
+The following list describes the key documentation features of ROSA:
+
+* **Getting started with ROSA**: ROSA 4.10 now includes a getting started guide. Getting Started with ROSA defines basic terminology and provides role-based next steps for developers and administrators. The tutorials walk new users through the web console and the OpenShift CLI interfaces. New users can accomplish the following tasks through the Getting Started:
+  ** Setting up the environment
+  ** Creating a ROSA cluster with STS using the default options
+  ** Creating a cluster administrator user for quick cluster access
+  ** Configuring an identity provider and granting cluster access
+  ** Accessing a cluster through the web console
+  ** Deploying an application from the Developer Catalog
+  ** Revoking administrator privileges and user access
+  ** Deleting a ROSA cluster and the AWS STS resources
+
+For more information, see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Getting started with {product-title}].
+
+* **Documentation restructure**: The ROSA product documentation went through two phases of restructuring focused on making the following more intuitive, clear, and findable:
+  ** Navigation menu
+  ** Book content
+  ** Topic flow
+
+In addition, redirects were put in place to help ensure a seamless navigation experience in the ROSA documentation.
+
+[id="technical-changes_{context}"]
+== Notable technical changes
+ROSA 4.10 introduces the following notable technical changes.
+
+[id="notable-changes-ephemeral-volumes"]
+=== Encrypted ephemeral instance volumes
+Clusters built in July 2022 and later use encrypted ephemeral instances as part of the pre-create cluster workflow. These will pass Service control policies (SCPs) that demand encrypted instance volumes only.
+
+[id="notable-changes-throttling-improvements"]
+=== Throttling improvements for ROSA CLI
+Delays, intervals and timeouts were improved to reduce the possibility of being throttled when addressing the AWS APIs.
+
+[id="notable-changes-new-commands"]
+=== Addition of new commands
+The following commands were added:
+
+* Managing a managed service / appliance: `Rosa create|edit managed-service --id=<service> --parameter-key <parameter value>`
+* Managing OCM roles: `Rosa create|list|delete ocm-role`
+* Managing OCM users: `Rosa create|list|delete user-role`
```

Comment:
```suggestion
The following ROSA CLI commands were added:

* Managing a managed service / appliance: `rosa create|edit managed-service --id=<service> --parameter-key <parameter value>`
* Managing OCM roles: `rosa create|list|delete ocm-role`
* Managing OCM users: `rosa create|list|delete user-role`
```

Please don't merge this commit directly, just adding a suggestion since I think these commands should be lowercase?

Reply:
Updated the commands